sdk/keyvault/keyvault-admin/src/backupClient.ts

```diff
@@ -0,0 +1,277 @@
+   * const poller = await client.beginRestore(blobStorageUri, sasToken, folderName);
+   *
+   * // Serializing the poller
+   * const serialized = poller.toString();
+   * // A new poller can be created with:
+   * // await client.beginRestore(blobStorageUri, sasToken, folderName, { resumeFrom: serialized });
+   *
+   * // Waiting until it's done
+   * const backupUri = await poller.pollUntilDone();
+   * console.log(backupUri);
+   * ```
+   * @summary Creates a new role assignment.
+   * @param blobStorageUri The URL of the blob storage resource, with the folder name of the blob where the previous successful full backup was stored.
+   * @param sasToken The SAS token.
+   * @param folderName The folder name of the blob where the previous successful full backup was stored.
+   * @param [options] The optional parameters.
+   */
+  public async beginRestore(
+    blobStorageUri: string,
+    sasToken: string,
+    folderName: string,
+    options: BeginRestoreOptions = {}
+  ): Promise<PollerLike<PollOperationState<undefined>, undefined>> {
+    if (!(blobStorageUri && sasToken && folderName)) {
+      throw new Error(
+        "beginRestore requires non-empty strings for the parameters: blobStorageUri, sasToken and folderName."
+      );
+    }
+
+    const poller = new RestorePoller({
+      blobStorageUri,
+      sasToken,
+      folderName,
+      client: this.client,
+      vaultUrl: this.vaultUrl,
+      intervalInMs: options.intervalInMs,
+      resumeFrom: options.resumeFrom,
+      requestOptions: options
+    });
+
+    // This will initialize the poller's operation (the generation of the backup).
+    await poller.poll();
+
+    return poller;
+  }
+
+  /**
+   * Starts restoring all key versions of a given key using user supplied SAS token pointing to a previously
+   * stored Azure Blob storage backup folder.
+   *
+   * This function returns a Long Running Operation poller that allows you to wait indefinitely until the Key Vault backup is generated.
+   *
+   * Example usage:
+   * ```ts
+   * const client = new KeyVaultBackupClient(url, credentials);
+   *
+   * const keyName = "<key-name>";
+   * const blobStorageUri = "<blob-storage-uri>";
+   * const sasToken = "<sas-token>";
+   * const poller = await client.beginSelectiveRestore(keyName, blobStorageUri, sasToken);
+   *
+   * // Serializing the poller
+   * const serialized = poller.toString();
+   * // A new poller can be created with:
+   * // await client.beginSelectiveRestore(keyName, blobStorageUri, sasToken, { resumeFrom: serialized });
+   *
+   * // Waiting until it's done
+   * const backupUri = await poller.pollUntilDone();
+   * console.log(backupUri);
+   * ```
+   * @summary Creates a new role assignment.
+   * @param keyName The name of the key that wants to be restored.
+   * @param blobStorageUri The URL of the blob storage resource, with the folder name of the blob where the previous successful full backup was stored.
```

Comment:
```suggestion
   * @param blobStorageUri The URL of the blob storage resource where the previous successful full backup was stored.
```

Reply:
I'm not applying this change because of https://github.com/Azure/azure-sdk-for-js/pull/11009#discussion_r483718133